debezium-server/debezium-server-bom/pom.xml

```diff
@@ -88,6 +90,11 @@
                 <artifactId>debezium-server-pravega</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-server-nats_streaming</artifactId>
```

Comment:
Could you please use dash instead of undersocre?

Reply:
Done!

Reply:
Done!